apps/chef_objects/src/chef_key_base.erl

```diff
@@ -57,7 +58,7 @@ maybe_generate_key_pair(Data) ->
 %% `public_key' field.
 %% If a private key is defined, then the private key will be placed in the `private_key'
 %% field.
--spec set_key_pair(ej:json_object(), {public_key, undefined | null| binary()}, {private_key, undefined | binary()}) -> ej:json_object().
```

Comment:
If undefined is no longer allowed/expected, do we need the case statement starting at line 64?